src/types/Schedule.ts

```diff
@@ -132,188 +148,219 @@ export const ScheduleMutation = extendType({
 
-        const sections = [
-          {
-            code: 'A01',
-            professorId: 1,
-          },
-          {
-            code: 'A01',
-            professorId: 2,
-          },
-          {
-            code: 'A01',
-            professorId: 3,
-          },
-        ];
-
-        const meetingTimes = [
-          [
-            {
-              day: 'TUESDAY',
-              startTime: new Date('2021-01-20T12:00:00.000Z'),
-              endTime: new Date('2021-01-20T12:50:00.000Z'),
-            },
-            {
-              day: 'WEDNESDAY',
-              startTime: new Date('2021-01-20T12:00:00.000Z'),
-              endTime: new Date('2021-01-20T12:50:00.000Z'),
-            },
-            {
-              day: 'FRIDAY',
-              startTime: new Date('2021-01-20T12:00:00.000Z'),
-              endTime: new Date('2021-01-20T12:50:00.000Z'),
-            },
-          ],
-          [
-            {
-              day: 'TUESDAY',
-              startTime: new Date('2021-01-20T13:00:00.000Z'),
-              endTime: new Date('2021-01-20T13:50:00.000Z'),
-            },
-            {
-              day: 'WEDNESDAY',
-              startTime: new Date('2021-01-20T13:00:00.000Z'),
-              endTime: new Date('2021-01-20T13:50:00.000Z'),
-            },
-            {
-              day: 'FRIDAY',
-              startTime: new Date('2021-01-20T13:00:00.000Z'),
-              endTime: new Date('2021-01-20T13:50:00.000Z'),
-            },
-          ],
-          [
-            {
-              day: 'MONDAY',
-              startTime: new Date('2021-01-20T13:00:00.000Z'),
-              endTime: new Date('2021-01-20T13:50:00.000Z'),
-            },
-            {
-              day: 'THURSDAY',
-              startTime: new Date('2021-01-20T13:00:00.000Z'),
-              endTime: new Date('2021-01-20T13:50:00.000Z'),
-            },
-          ],
-        ];
+          // Update capacities for the new courses from algorithm 2's response
+          courseCapacities.forEach(({ code, subject, semester, capacity }) => {
+            const courseIndex = newCourses.findIndex(
+              (course) => code === course.code && subject === course.subject && semester === course.term
+            );
+            newCourses[courseIndex].capacity = capacity;
+          });
 
-        const s = await (prisma as PrismaClient).schedule.create({
-          data: {
-            createdAt: new Date(),
-            year,
-          },
-        });
+          // Create course input for algorithm 1
+          const algo1Courses: Algo1Course[] = [];
+          newCourses.forEach(({ code, subject, capacity, sectionCount: numSections }) => {
+            algo1Courses.push({
+              courseNumber: code,
+              subject,
+              sequenceNumber: 'A01',
+              streamSequence: '2A',
+              courseTitle: 'Calculus',
```

Comment:
Need to ask about stream sequence cause it seems to have no effect. Sequence number also doesn't matter because we pass in the # of sections which algo1 reads and makes new A01, A02, A03, etc. based on that. No idea where we would be getting course title from unless we make a table in the DB that ties `subject` and `code` to a `title` and then seed it.

Reply:
I'll add this as a comment for clarity